translations/ui-invoice/en.json

```diff
@@ -218,6 +218,7 @@
   "invoice.details.information.billToName": "Bill to name",
   "invoice.details.information.billToAddress": "Bill to address",
   "invoice.details.information.batchGroup": "Batch group",
+  "invoice.details.information.note": "Note",
```

Comment:
Does it mean that field has no label on the form?

Reply:
No)